src/lib/room.js

```diff
@@ -201,16 +201,18 @@ export const loadMessages = async () => {
 	await store.setState({ messages: (messages || []).reverse(), noMoreMessages: false, loading: false });
 
 	if (messages && messages.length) {
-		const lastMessage = messages[messages.length - 1];
-		await store.setState({ lastReadMessageId: lastMessage && lastMessage._id });
-
-		if (lastMessage.t === constants.webrtcCallStartedMessageType || lastMessage.t === constants.jitsiCallStartedMessageType) {
-			if (lastMessage.endTs) {
-				await store.setState({ ongoingCall: { callStatus: 'ended', time: lastMessage.ts }, incomingCallAlert: null });
-				return;
+		for (let i = 0; i < messages.length; ++i) {
```

Comment:
IMO, we should be relying on the status property on room object instead of looping through all the messages - to check if any call is still on-going

Reply:
According to your point, after refresh the call status property should be dependent on room object instead of looping through all the messages which I will be fixing in my new pr( call declined message at livechat widget). @murtaza as this pr is for webrtc call for mobile-device and for mobile device this pr is working good as expected. I think we can move forward.